CMakeLists.txt

```diff
@@ -2350,6 +2355,11 @@ elseif(APPLE)
 
   # Actually load the frameworks at the end so we don't duplicate include.
   if(SDL_FRAMEWORK_COREVIDEO)
+    find_library(COREMEDIA CoreMedia)
+    if(COREMEDIA)
+      set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -DHAVE_COREMEDIA=1")
```

Comment:
Perhaps add `#cmakedefine HAVE_COREMEDIA` to `SDL_buildconfig.h.cmake` + other build configs, and use it with `#ifdef HAVE_COREMEDIA`

(If you agree, I can do that)